eif_utils/src/lib.rs

```diff
@@ -382,6 +382,59 @@ impl<T: Digest + Debug + Write + Clone> EifBuilder<T> {
         }
     }
 
+    /// Get metadata parts, add them to the structure and
+    /// serialize all the information as JSON
```

Comment:
Is the JSON serialized in a reproducible manner? Fields should be sorted alphabetically, etc.

Reply:
It depends on the implementation of `serde_json`. I guess it should be reproducible, but I need to check.

Reply:
From what I understood, by default `serde_json` sorts the fields alphabetically as you said, until you import it with `preserve_order` feature enabled. That only works for `serde:json::Value` objects as for structs they are represented as a `Map` of specific type (depending on `preserve_order` flag).
Thus, if we have serializable `let eif_info EifIdentityInfo` struct, it can be serialized with alphabetical order of fields the following way:
```
serde_json::to_vec(serde_json::to_value(eif_info))
```
While we currently have it like this
```
serde_json::to_vec(eif_info)
```
and it will **not** sort the fields alphabetically.

The question here, is it important for metadata to have a specific order considering it's not affecting the PCR values?